src/ts/types/modules.d.ts

```diff
@@ -1,3 +1,8 @@
+declare module '*.svg' {
+  const content: string
+  export default content
+}
```

Comment:
Insane nitpick but the longer type declaration should go beneath the shorter one.